config/prow/plugins.yaml

```diff
@@ -484,67 +484,51 @@ repo_milestone:
     # You can curl the following endpoint in order to determine the github ID of your team
     # responsible for maintaining the milestones. You may need to specify the page number.
     # curl -H "Authorization: token <token>" "https://api.github.com/orgs/<org-name>/teams?page=N"
-    maintainers_id: 2460384
```

Comment:
somebody with more prow knowledge should confirm this is safe to remove

Reply:
its in the documentation.... but yea probably a good idea to double check with someone

Reply:
While deprecated, `maintainers_id` is a valid field: https://github.com/kubernetes/test-infra/blob/682727078f0c86e556d9d766bd4ffbf3e2980a56/prow/plugins/config.go#L481

Reply:
is it okay to be taken out then?